src/modules/users/user_repository.ts

```diff
@@ -23,20 +24,28 @@ export class UserRepository {
         'users.updated_at',
         'users.last_login_at',
         'files.name as file_name',
-        'files.id as file_id'
+        'files.id as file_id',
+        'partners.name as partner_name'
       )
       .leftJoin('files', 'files.source', '=', 'users.avatar')
+      .leftJoin('partners', 'partners.id', '=', 'users.partner_id')
 
   public findAll = (request: UserEntity.RequestQuery) => {
-    const orderBy: string = request.order_by || 'updated_at'
+    const orderBy: string = request.order_by || 'users.updated_at'
     const sortBy: string = request.sort_by || 'desc'
 
     const query = this.Query().orderBy(orderBy, sortBy)
 
     if (convertToBoolean(request.is_active))
       query.where('users.is_active', convertToBoolean(request.is_active))
 
-    if (request.is_admin) query.where('users.is_admin', convertToBoolean(request.is_admin))
+    if (request.is_admin) {
+      query.where('users.is_admin', convertToBoolean(request.is_admin))
+    }
+
+    if (request.roles && request.roles === config.get('role.1')) {
+      query.whereNotNull('users.partner_id')
+    }
```

Comment:
bisa dikasih comment kang @ayocodingit untuk pengkondisian ini untuk apanya?

Reply:
siap udh di kasih yah kang @rindibudiaramdhan disini [add comment condition filter roles value is partner](https://github.com/jabardigitalservice/dedi-core-service/pull/193/commits/23702f36558c87c21a84d26f26d965c923999a90)

Reply:
sip mantul kang @ayocodingit !
berarti ada penambahan env baru ya? untuk `config.get('role.1')`

Reply:
udh ada di env nya mah kang, cuman untuk list users mitra ini aku gunain satu endpoint aja, menghandle semua roles, makanya akan banyak kondisi untuk bisa menangani nya